src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -345,6 +345,50 @@ export const getOVMethodTypeAuthenticatorButtonElements = (
   return reorderAuthenticatorButtons(authButtons, deviceKnown);
 };
 
+export const getCustomAppMethodTypeAuthenticatorButtonElements = (
```

Comment:
This function looks almost identical to the [one](https://github.com/okta/okta-signin-widget/pull/3277/files#diff-12d966f9da6aa08edf71323461a36f678d957daac3430118bde74fd8fce7130bR306) above. I would instead update the existing function that accepts a param of the authenticator key. It would also require the function name to change but reduce the duplicated lines between these two functions.